modules/doc/content/newsletter/2023/2023_05.md

```diff
@@ -22,20 +17,90 @@ Several `Positions` were implemented to help kickstart the use of the system:
 
 When more than one time stepper is provided, the new time stepper system will automatically compute the minimum time step among all the input time steppers as the time step size to use for that application. The new time stepper system also provides a [!param](/Executioner/TimeSteppers/lower_bound) parameter to further control the time stepper size (e.g. enable growth driven by a single time stepper). This system is controllable via the [Controls](syntax/Controls/index.md) system. The user can turn on/off the provided time steppers to control the usage of time steppers ; for example, one could make time stepper(s) only active [during a certain time period](TimePeriod.md).
 
-## libMesh-level Changes
+## Modules improvements
+
+### Fluid properties for the NaK eutectic
+
+Fluid properties were added for the `NaK` eutectic in the fluid properties module.
+These are extracted from the public Sodium handbook chapter on fluid properties \cite().
+The fluid properties implemented should suffice for use with the Navier Stokes module.
 
-## PETSc-level Changes
+### Functor solid thermal properties
+
+A functor version of the solid thermal properties material has been added to the solid properties module.
+Functor materials are more flexible, enabling all types of arbitrary evaluations on faces or points with an element for example.
+This addition is leveraged by the finite volume heat conduction solver.
 
 ## Bug Fixes and Minor Enhancements
 
+- Moving source points in Dirac Kernels, such as the [ReporterPointSource.md] are now supported.
+- A [ParsedVectorAux.md] has been added to MOOSE. It enables forming in a very
+  flexible manner vector auxiliary variables. It can be used for initialization
+  of these variables, a long standing limitation of using vector variables.
+- Added error checking for calling variable APIs that would result in use of
+  `Assembly` neighbor AD data which has not been implemented. Consequently the
+  neighbor test gradient in `ADInterfaceKernel` no longer pretends to have
+  derivative data.
+- Advection schemes other than `upwind` and `average` can now be used in
+  `FVAdvection`, e.g. TVD schemes such as `min_mod` and `vanLeer`. This is
+  useful for users who want to model scalar advection in a constant background
+  velocity field.
+- Dirichlet finite volume boundary conditions can now be controlled using the
+  [Controls](syntax/Controls/index.md) system.
+- We now always use undisplaced neighbor reference points when reinitializing
+  displaced mesh neighbor finite element data. This is both an optimization and
+  a bug fix for contexts like cohesive zones in which attempting to invert the
+  physical points from the element side into reference points on the neighbor side would
+  fail because the element and neighbor sides are no longer coincident.
```

Comment:
```suggestion
- Undisplaced neighbor reference points are now always used when reinitializing
  displaced mesh neighbor finite element data. This is both an optimization and
  a bug fix for contexts like cohesive zones, in which attempting to invert the
  physical points from the element side into reference points on the neighbor side would
  fail because the element and neighbor sides are no longer coincident.
```